src/bin.js

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env -S XDG_DATA_DIRS=${XDG_DATA_DIRS}:/run/host/usr/share:/var/lib/snapd/desktop:/var/lib/flatpak/exports/share:${HOME}/.local/share/flatpak/exports/share gjs -m
+#!/usr/bin/env -S XDG_DATA_DIRS=${XDG_DATA_DIRS:-/usr/local/share:/usr/share}:/run/host/usr/share gjs -m
```

Comment:
Thanks for the quick response! If you are referring to the `${var:-default}` shell syntax, I hope this explains it:

From the Bash manpage:
> ${parameter:-word}
>               Use Default Values.  If parameter is unset or null, the expansion of word is substituted.  Otherwise, the value of parameter is substituted.

Reply:
@sonnyp Can I help you with anything else? :)

Reply:
> Thanks for the quick response! If you are referring to the `${var:-default}` shell syntax, it substitutes the default value `default` if `${var}` is unset or null:
> 
> From the Bash manpage:
> 
> > ${parameter:-word}
> > Use Default Values.  If parameter is unset or null, the expansion of word is substituted.  Otherwise, the value of parameter is substituted.

@antecrescent 

Something doesn't work with this syntax, the following error is received:
```
$ re.sonny.Junction
/usr/bin/env: only ${VARNAME} expansion is supported, error at: ${XDG_DATA_DIRS:-/usr/local/share:/usr/share}:/run/host/usr/share gjs -m
```

Maybe env (at least on my arch) does not support this `${parameter:-word}` shell syntax.